Makefile

```diff
@@ -0,0 +1,10 @@
+proto_gen:
+	protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative pkg/grpc/proto/hackernews.proto
+
+proto_mock:
+	mockgen -package=proto -source=pkg/grpc/proto/hackernews_grpc.pb.go -destination=pkg/grpc/proto/mockClient.go
+
+proto: proto_gen proto_mock
+
+test_int_coverage:
```

Comment:
Personal preference, but might want to break this one command into a few commands and call them from one overall method e.g. 
```

lint.go.fmt:
	go fmt ./...;

lint.go.vet:
	go vet ./...;
	
lint.go: lint.go.fmt lint.go.vet
```

Reply:
I only that command so it wouldn't be in my bash history only. That's mostly a helpful command to run locally before committing code to check code coverage.

Each of the commands in `test_int_coverage` have a dependency on the one before